### PR TITLE
fetched data from api and created component EmployeeList

### PR DESCRIPTION
**Pull request by AdriannaLen**

### CODE REVIEW

**Comment by AdriannaLen** [comment]
Added fetch request from API and created component Employee list to display employees

File: src/Context/AppContext.tsx [hunk 1]
```diff
@@ -1,4 +1,39 @@
-export {};
+import { createContext } from "react";
+import useFetchEmployees, { Employee } from "./hooks/useAppContext";
+
+type EmployeeContextType = {
+  employees: Employee[];
+  isLoading: boolean;
+};
+
```
Comment (szu-szy) [inline src/Context/AppContext.tsx:8]: please format this file

File: src/Context/AppContext.tsx [hunk 2]
```diff
@@ -1,4 +1,39 @@
...
+      >
+        {children}
+      </AppContext.Provider>
+    );
+  };
+
+export default AppContextProvider;
+
 
 // // krok 1 - tworzenie typu kontekstu
```
Comment (szu-szy) [inline src/Context/AppContext.tsx]: please remove unnecessary comments

File: src/Context/hooks/useAppContext.tsx [hunk 3]
```diff
@@ -1 +1,55 @@
-export {};
```
Comment (szu-szy) [inline src/Context/hooks/useAppContext.tsx]: please format file

File: src/Context/hooks/useAppContext.tsx [hunk 4]
```diff
@@ -1 +1,55 @@
...
+        }
+
+        finally {
+            setIsLoading(false);
+        }
+    }
+
+    useEffect(() => {
+        fetchEmployees();
+
```
Comment (szu-szy) [inline src/Context/hooks/useAppContext.tsx]: please remove empty line

File: src/components/EmployeeList/EmployeeList.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+import  { AppContext } from "../../Context/AppContext"
+
+const EmployeeList = () => {
+
+    const { employees } = useContext(AppContext);
+
+    return (
+        <div>
+            <ul>
+                {employees.map((user) => (
```
Comment (szu-szy) [inline src/components/EmployeeList/EmployeeList.tsx]: if u are using - user.id, user.firstName, user.lastName you could destructure this object

File: src/pages/Home/index.tsx [hunk 6]
```diff
@@ -1,8 +1,12 @@
 import React from "react";
 import "./index.scss";
+import EmployeeList from "../../components/EmployeeList/EmployeeList";
 
 const Home = () => {
-  return <div>HOME</div>;
+  return <div>
```
Comment (szu-szy) [inline src/pages/Home/index.tsx:6]: if it possible please remove div and attach better tag

**Pull request merged**